tests/writer_test.py

```diff
@@ -59,9 +60,14 @@ def testYamlWriter(self):
     file_object = io.StringIO(initial_value=artifacts_yaml)
     converted_artifact_definitions = list(artifact_reader.ReadFileObject(
         file_object))
-    self.assertCountEqual(
-        [artifact.AsDict() for artifact in artifact_definitions],
-        [artifact.AsDict() for artifact in converted_artifact_definitions])
+    if sys.version_info >= (3, 0):
```

Comment:
What issue are you trying work around here? You can likely solve this in a different way.